app/controllers/maintenance_tasks/tasks_controller.rb

```diff
@@ -21,6 +21,26 @@ def show
       @task = TaskData.find(params.fetch(:id))
       set_refresh if @task.last_run&.active?
       @runs_page = RunsPage.new(@task.previous_runs, params[:cursor])
+      @is_creator = @task.last_run&.creator.present? && @task.last_run&.creator == current_user_identifier
```

Comment:
`current_user_identifier` should be implemented by the hosting app. A default implementation must be added to the engine.